saleor/app/installation_utils.py

```diff
@@ -73,6 +74,8 @@ def install_app(app_installation: AppInstallation, activate: bool = False):
         type=AppType.THIRDPARTY,
         audience=manifest_data.get("audience"),
     )
+
+    app.store_value_in_private_metadata({"pending_installation": True})
```

Comment:
This is fine, but why not store it in the model as a regular field?

Reply:
I didn't want to make the model messy, as this is needed for couple of seconds. Any pros for regular field in this case?

Reply:
The field on the model will be a better solution. (If we introduce filtering via this field, it's cheaper to optimize it.)

Reply:
Generally, model fields are better for filtering and DB operations, while metadata is mostly for external integrations to write arbitrary data we don't need to know about. We have the AppInstallation model, which is dedicated to tracking the installation process, could we store this info there perhaps?

Reply:
We filter through this field in resolvers. For sure we should add a field on the model for storing this information.

Reply:
OK, so I will add field to App model, because AppInstallation has no relationship with App